src/Utils.cc

```diff
@@ -182,7 +182,7 @@ void throwOrPrintErrors(const sdf::Errors& _errors)
 /// tree.
 /// \param[in] _sdf sdf::ElementPtr of the entity with the name attribute
 /// \param[out] _errors Will contain errors encountered in the function.
-/// \return Absolute name of the entity of no errors occured. nullopt otherwise.
+/// \return Absolute name of the entity of no errors occurred. nullopt otherwise.
```

Comment:
```suggestion
/// \return Absolute name of entity if no errors occurred, nullopt otherwise.
```
suggestion to fix typo and reduce line length to fix linter warning

Reply:
fixed already in https://github.com/gazebosim/sdformat/pull/1552/commits/e9b9440d803e560918a2e5cf682c4523a84d8257